charts/zot/Chart.yaml

```diff
@@ -3,4 +3,5 @@ appVersion: v2.1.2
 description: A Helm chart for Kubernetes
 name: zot
 type: application
-version: 0.1.66
```

Comment:
This needs fixing?

Reply:
What version is expected?

Remove `home` then?

Reply:
next version should be 0.1.67

https://artifacthub.io/packages/helm/zot/zot

Reply:
and yes remove "home", that happens automatically

Reply:
Thanks for the feedbacks @rchincha , just pushed the changes.